CHANGELOG.md

```diff
@@ -3,14 +3,8 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 
 ## Unreleased
 
-### Added
-- 
-
 ### Changed
-- 
-
-### Removed
-- 
+- Normalize markdown formatting across usage.md files
```

Comment:
@Scotchester would we leave `All components` there for release? Or is that just there to remind us to list every individual component come release time?